Projects/Server/Items/Container.cs

```diff
@@ -34,6 +36,7 @@ namespace Server.Items
 
   public class Container : Item
   {
+    private static QueuePool m_QueuePool = new QueuePool(QueueRef<Container>.Generate, preGenerateCount: 2, maxRefrenceRetention: 5);

```

Comment:
I wonder how we can know if these numbers are good. Thoughts?

Reply:
Given that we are single threaded and non recursive, I doubt we actually will use more just one. The only way this would grow is if the container spins up parallel searches, or if the Queue Pool is shared with other classes that use queues while also calling into the search.

Reply:
That is a good point. I'll think this over. It is possible we don't need this and we can just have a single static reference.
I like this though, we may use it somewhere else.